client/gutenberg/extensions/presets/jetpack/editor-shared/get-gutenberg-site-fragment.js

```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSiteFragment } from 'lib/route/path';
+
+/***
+ * Returns the site fragment in the environment we're running Gutenberg in.
+ *
+ * @returns {?Number|String} Site fragment (ID or slug); null for WP.org wp-admin.
+ */
+export default function getGutenbergSiteFragment() {
```

Comment:
I'm not opposed to having `gutenberg` here and I don't want to start a bikeshed; just noting that maybe more generic `getBlockEditorSiteFragment` would work just as well so that we don't end up having `gutenberg` in every method that touches the editor. :-)

Reply:
Or just `getSiteFragment`? (We can alias the import from `lib/route/path`.)

Reply:
`getSiteFragment` it is 👍 https://github.com/Automattic/wp-calypso/pull/28998/commits/7ab762fc557df18347ee823414e637147fdd972c